core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -244,7 +249,8 @@ class KafkaApis(val requestChannel: RequestChannel,
               s"on partition $topicPartition failed due to ${Errors.forCode(errorCode).exceptionName}")
```

Comment:
We should also log information about the unauthorized for describe with debug enabled, probably.